pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -135,6 +136,21 @@ type PrefilterPlugin interface {
 	Prefilter(pc *PluginContext, p *v1.Pod) *Status
 }
 
+// FilterPlugin is an interface for Filter plugins. These plugins are called at the
+// filter extension point for filtering out hosts that cannot run a pod.
+// This concept used to be called 'predicate' in the original scheduler.
+// These plugins should return only Success or Error in Status.code. However,
+// the scheduler accepts other valid codes as well. Anything other than Success
+// will lead to exclude hosts from running the pod.
+type FilterPlugin interface {
+	Plugin
+	// Filter is called by the scheduling framework.
+	// All FilterPlugins should return sucecss. Otherwise, the node is filtered out.
+	// If Filter returns error, please refer scheduler/algorithm/predicates/error.go
+	// to set error message.
```

Comment:
update the comment to explicitly say that a filter plugin should return success to declare that the node fits, or unschedulable to indicate that the node does not fit. Other statuses are considered internal errors.

Reply:
Addressed.